test-config/sensirion_test_setup.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SENSIRION_TEST_SETUP_H
+#define SENSIRION_TEST_SETUP_H
+
+#include <stdio.h>
+
+#include "CppUTest/TestHarness.h"
+
+#include "i2c_mux.h"
+
+#define CHECK_ZERO(actual) CHECK_EQUAL(0, (actual))
+#define CHECK_ZERO_TEXT(actual, text) CHECK_EQUAL_TEXT(0, (actual), (text))
```

Comment:
I try to get these upstream: https://github.com/cpputest/cpputest/pull/1423

Reply:
Okay can you merge it when it is ready?

Reply:
Removed them.

Reply:
Let's replace these macros with the up-streamed ones in a follow up. We'll probably wait until a new version of cpputest is released which includes these.